site/source/user-manual/service-guides/vaultwarden/bitwarden-client-setup.rst

```diff
@@ -153,10 +177,14 @@ iOS
                 :width: 50%
                 :alt: vaultwarden-tor-address
 
-        #. Now send that address to your phone and paste it into Bitwarden but **before you hit save:**
+        #. Now send that address to your phone and paste it into Bitwarden.
 
-            .. note:: The Tor address you will have copied will begin with **http** - Please change this to **https** instead of **http**
+            .. note::  **before you hit save:** The Tor address you will have copied will begin with **http** - Please change this to **https** instead of **http**
```

Comment:
```suggestion
            .. caution::  **Before you hit save:**  The Tor address you will have copied will begin with **http** - Please change this to **https** instead of **http**
```

Reply:
Much better.